kmapper/kmapper.py

```diff
@@ -750,6 +754,7 @@ def visualize(
         mapper_data = format_mapper_data(
             graph,
             color_function,
+            colorscale,
```

Comment:
pass colorscale to `format_mapper_data` to get colorscale into cluster details view